ding/utils/logging_rich_config.py

```diff
@@ -1,6 +1,81 @@
+import platform
+import os
+import sys
+from typing import Optional
 import logging
 from rich.logging import RichHandler
+from rich import console
 
-logging.basicConfig(
-    level=logging.INFO, format="%(message)s", datefmt="[%Y-%m-%d %H:%M:%S]", handlers=[RichHandler()], force=True
-)
+
+def enable_rich_handler(level: int = logging.INFO) -> None:
+    r'''
+    Overview:
+        Enable rich handler decoration to logger. Default logging.StreamHandler will be replaced.
+    Arguments:
+        - level (:obj:`int`): Logger Level for Rich handeler, default set to ``logging.INFO``.
```

Comment:
use proper newline notation for these comment, you can refer to `ding/model/template/qac.py`

Reply:
Thanks, it helps a lot. Newline notation works great!